### PR TITLE
Implement PHP Lint checks

### PR DESCRIPTION
**Pull request by ArjenMiedema**
The PHPLint task will check your source files for syntax errors and
has been implemented in the testing suite now.

### CODE REVIEW

File: config/default/grumphp.yml [hunk 1]
```diff
@@ -22,6 +22,12 @@ parameters:
   phpcs.standard: ./phpcs.xml
   phpcs.triggered_by: [php]
 
+  phplint.exclude: [ ]
```
Comment (leonhelmus) [inline config/default/grumphp.yml:25]: Please add vendor here
See this comment of phplint: https://github.com/phpro/grumphp/blob/master/doc/tasks/phplint.md
Any directories to be excluded from linting. You can specify which directories you wish to exclude, such as the vendor directory.

Comment (ArjenMiedema) [reply config/default/grumphp.yml]: @leonhelmus this has been updated.

**Review by ksangers — APPROVED** [review]

**Comment by ksangers** [comment]
@ArjenMiedema Could you please resolve the conflicts so that rebase becomes available? Base your edits on the latest master branch commit.

**Comment by ArjenMiedema** [comment]
@ksangers conflicts have been resolved.

**Comment by leonhelmus** [comment]
Will merge it next week :).

**Pull request merged**